packages/terra-form/src/_Utilities.jsx

```diff
@@ -0,0 +1,37 @@
+const autoResizeTextareaHeight = (onChange, baseRowCount) =>
+  (e) => {
+    const textarea = e.currentTarget;
+    const lineHeight = Math.ceil(parseFloat(window.getComputedStyle(textarea).lineHeight, 0));
+
+    /* Smallest Textarea should have at least 2 rows. */
+    const minRows = baseRowCount || 2;
+    textarea.rows = minRows;
+    const rows = Math.ceil((textarea.scrollHeight - textarea.baseScrollHeight) / lineHeight);
+    textarea.rows = minRows + rows;
+
+    if (onChange) {
+      onChange(e);
+    }
+  };
+
+const setTextareaBaseHeight = onFocus =>
+  (e) => {
```

Comment:
again can spell out what e is

Reply:
Spelled out here. https://github.com/cerner/terra-core/pull/966/commits/4a949d8e4efe5e688a7fc02f51f8662e4e96bae3